src/main/java/com/digitax/controller/RelationController.java

```diff
@@ -23,10 +25,14 @@
 	 RelationRepository relationRepository;
 	 
 	 
-	    @GetMapping("/get/relations")
+	    @SuppressWarnings("unchecked")
+		@GetMapping("/get/relations")
 	    public ResponseEntity<?>  getRelations(){
 	    	ArrayList<Relation> obj = new ArrayList<>(relationService.getRelation()); 
-			 return new ResponseEntity<>(ApiRes.success(obj,200).setMessage("List Relation"), HttpStatus.OK);
+	    	JSONObject statusObj=new JSONObject();    
+			statusObj.put("status_code",200);  
```

Comment:
@Jayanta-Redapple This is a perfect example of how creating Constants for these keys is beneficial. You can reference the same Constant in multiple classes and instances. Lets update this to use Constants and avoid any hardcoded values.